lib/elixir/lib/kernel.ex

```diff
@@ -3409,7 +3409,7 @@ defmodule Kernel do
 
   """
   defmacro to_charlist(term) do
-    quote(do: List.Chars.to_charlist(unquote(term)))
+    quote(do: :"Elixir.List.Chars".to_charlist(unquote(term)))
```

Comment:
Can you elaborate on this?

Reply:
Just for consistency with to_string above, it makes the rewriting back easier (since we don't need to match on aliases).